snapshot/snapshot.go

```diff
@@ -301,21 +301,19 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 
 	if info.Kind == snapshots.KindActive {
 		pKey := info.Parent
-		pID, info, _, err := snapshot.GetSnapshotInfo(ctx, o.ms, pKey)
-		if err != nil {
-			return nil, errors.Wrapf(err, "get snapshot %s info", pKey)
-		}
-
-		if label.IsNydusMetaLayer(info.Labels) {
-			err = o.fs.WaitUntilReady(pID)
-			if err != nil {
-				return nil, errors.Wrapf(err, "mounts: snapshot %s is not ready, err: %v", pID, err)
+		if pID, info, _, err := snapshot.GetSnapshotInfo(ctx, o.ms, pKey); err == nil {
```

Comment:
We should handle the error other than `errdefs.NotFound`.

Reply:
Good point. Will fix it.